crispy_tailwind/templatetags/tailwind_field.py

```diff
@@ -114,9 +114,12 @@ def render(self, context):  # noqa: C901
             attrs = [attrs] * len(widgets)
 
         converters = {
-            "textinput": "textinput textInput",
+            "textinput": "bg-white focus:outline-none border border-gray-300 rounded-lg py-2 px-4 block w-full \
+                appearance-none leading-normal",
             "fileinput": "fileinput fileUpload",
             "passwordinput": "textinput textInput",
+            "emailinput": "bg-white focus:outline-none border border-gray-300 rounded-lg py-2 px-4 block w-full \
```

Comment:
Can we add these classes to the `default_styles` dict on the class instead?

Will need to drop the trailing input/widget from from the widget type.